### ISSUE TITLE
Release 1.1.0 to Maven central

### ISSUE THREAD

**Issue by tinchodias** [uqbar-project/xtrest#16]

**Comment by tinchodias** [comment]
Trying to understand why https://travis-ci.org/uqbar-project/xtrest/builds/355680395 didn't release.

It ends with:

```
[INFO] Building jar: /home/travis/build/uqbar-project/xtrest/target/xtrest-1.1.2-javadoc.jar
[INFO] 
[INFO] --- maven-gpg-plugin:1.5:sign (sign-artifacts) @ xtrest ---
gpg: no default secret key: secret key not available
gpg: signing failed: secret key not available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 37.259 s
[INFO] Finished at: 2018-03-20T03:23:23Z
[INFO] Final Memory: 58M/630M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-gpg-plugin:1.5:sign (sign-artifacts) on project xtrest: Exit code: 2 -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

Don't know why is "secret key not available"...

**Comment by tinchodias** [comment]
Finally worked: the travis build published 1.1.3. Sadly 1.1.0, 1.1.1 and 1.1.2 were not published (the code is the same, only travis configuration changed). I'm sorry!

To make it work, I reverted a commit I did some months ago trying to fix a ci error in pull requests... but I made mistakes :-1:
